src/test/java/uk/gov/ons/ctp/integration/fieldsvccucumber/glue/RunCucumberTestSSO.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.ons.ctp.integration.fieldsvccucumber.glue;
+
+import io.cucumber.junit.Cucumber;
+import io.cucumber.junit.CucumberOptions;
+import org.junit.runner.RunWith;
+
+@RunWith(Cucumber.class)
+@CucumberOptions(
+    plugin = {"pretty", "html:target/cucumber"},
+    features = {"src/test/resources/features/sso"},
```

Comment:
while you are here , and renaming directories, would it be possible to remove the annoying "sso" folder under features ?  Would also need to change the directory for the actual feature file!

Reply:
Don't know why we have this - to be honest